lib/registration/disparity_registration.h

```diff
@@ -0,0 +1,25 @@
+/// \file
+/// Maintainer: Felice Serena
+///
+
+#pragma once
+
+#include "registration.h"
+
+namespace MouseTrack {
+
+class DisparityRegistration : public Registration {
+public:
+    PointCloud operator()(const FrameWindow& window) const;
+private:
+    /// constant from fpga set up
+    double min_disparity = 1*2+32;
+    /// constant from fpga set up
+    int xshift = 22;
+    /// constant from fpga set up
+    int yshift = -8;
+    /// constant from fpga set up
+    int frame_border = 80;
```

Comment:
Couldn't we make these four variables const? It's probably not important but just to be consistent.

Reply:
I actually tend to write accessors for them, since they are part of the set up, we/clients might want to change the values in the future.

Reply:
I probably also should adjust the description.